### DISCUSSION TITLE
How to Spell check repositories?

### DISCUSSION THREAD

**Discussion by opticaldrive** [community/community#61106] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to Spell check repositories?
I think I saw some repos with automatic spell checking.
How can I add that to my repos?
Sorry if this is the wrong category.
Thank you.

**Comment by the-officialjosh** [comment]
It would been better you name the repo so I can access and help you

**Reply by opticaldrive** [reply]
Pretty much all my repos.
But you can try with my test repo:
https://github.com/V205Arduino/Test

**Reply by opticaldrive** [reply]
Thank you.

**Reply by the-officialjosh** [reply]
No, I meant you show me a sample repo with the spell check.

**Reply by opticaldrive** [reply]
Ok!

**Reply by opticaldrive** [reply]
Can’t seem to find it….. :(

**Reply by opticaldrive** [reply]
https://github.com/marketplace/actions/github-spellcheck#
Is that free?
Will it work?

**Reply by opticaldrive** [reply]
Thank you.

**Comment by GhostOf0days** [comment]
Can you please try [this](https://github.com/marketplace/actions/check-spelling) and [this](https://github.com/tbroadley/github-spellcheck-cli)?

**Reply by opticaldrive** [reply]
are those free?
Are they trusted?
Or should i use GitHub’s one?
Thank you.

**Reply by GhostOf0days** [reply]
Yes. They are free. As for trusted, anything in the Marketplace is verified by Github. Also, for the other one, the repo is public, so you can view the source code. So the answer to both questions is yes. I haven't looked deep into them, but the Github Marketplace one seems more focused on making sure your variables are correctly spelled while the other one seems more general.

**Reply by opticaldrive** [reply]
So for general i should use:
https://github.com/marketplace/actions/github-spellcheck
Thanks.

**Reply by GhostOf0days** [reply, marked as answer]
That looks fine.

**Reply by opticaldrive** [reply]
Will do! Thank you!

**Reply by GhostOf0days** [reply]
No problem.

**Comment by opticaldrive** [comment]
Solved. Thank you.
Will add.
Have not added it yet.
